core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -749,6 +749,13 @@ class EMConfig {
     var extraHeuristicsFile = "extra_heuristics.csv"
 
 
+    @Cfg("Enable to print snapshots of the generated tests during the search in an interval defined in snapshotsInterval.")
+    var enableWriteSnapshotTests = false
+
+    @Cfg("The size (in seconds) of the interval that the snapshots will be printed, if enabled.")
+    var writeSnapshotTestsIntervalInSeconds = 3600
```

Comment:
add comment on this line like `// ie, 1 hour`

Reply:
also, would be good to have a E2E in which this value is set to something very short, eg 1 sec, to force the execution of this new code. not sure if there is an easy assertions to write here (eg check if files are generated / updated), but at least we should check that EM does not crash when this snapshot are generated